### PR TITLE
Append tookit version to buildPlugin zips

### PR DESCRIPTION
**Pull request by abrooksv**
```
ll */build/distributions/*
intellij/build/distributions/aws-toolkit-jetbrains-1.26-SNAPSHOT-202.zip
jetbrains-core/build/distributions/aws-toolkit-jetbrains-1.26-SNAPSHOT-202.zip
jetbrains-rider/build/distributions/aws-toolkit-jetbrains-1.26-SNAPSHOT-202.zip
jetbrains-ultimate/build/distributions/aws-toolkit-jetbrains-1.26-SNAPSHOT-202.zip

```

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -14,10 +14,6 @@ plugins {
     id("toolkit-jacoco-report")
 }
 
-group = "software.aws.toolkits"
```
Comment (rli) [inline build.gradle.kts:17]: group?

**Review by rli — APPROVED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (abrooksv) [reply build.gradle.kts:17]: We never used it it seems

I can add it for the sake of it though?

**Review by hunterwerlla — APPROVED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (abrooksv) [reply build.gradle.kts:17]: To be clear, we didnt use it because we dont publish anything

**Pull request merged**